orderer/consensus/smartbft/util_network_test.go

```diff
@@ -64,6 +64,7 @@ type NetworkSetupInfo struct {
 	genesisBlock *cb.Block
 	tlsCA        tlsgen.CA
 	configInfo   *ConfigInfo
+	Logger       *flogging.FabricLogger
```

Comment:
as a general rule of thumb - we should not couple this logger field with a specific implementation, but with an interface.